src/AITS/PersonLookupConfidential.php

```diff
@@ -171,6 +176,16 @@ public function getEmail(): string
         return $this->email;
     }
 
+    public function getEmployee(): array
+    {
+        return $this->employee;
+    }
+
+    public function getOrganizationDescription(): string
```

Comment:
@hualiu6 The return type here should be `?string` to indicate it is nullable, since it can be null when not for an employee.  @thedanpaz I'm pretty sure this is safe since it the minimum version of this package is php 7.2 and nullable types are php 7.1.

Reply:
@jjonesuic Update the default null to empty string, it should be ok.